### PR TITLE
Add mdns entries to openssh and mpd

### PR DESCRIPTION
**Pull request by neheb**

### CODE REVIEW

**Comment by thess** [comment]
These updates change the binary contents of the related packages - please bump PKG_RELEASE number for each.

File: sound/mpd/files/mpd.init [hunk 1]
```diff
@@ -19,6 +19,7 @@ start_service() {
 	fi
 
 	procd_open_instance
+	procd_add_mdns "mpd" "tcp" "6600"
```
Comment (yousong) [inline sound/mpd/files/mpd.init]: It will be convenient if the port can be decided by checking `CONFGIFILE`.

**Comment by neheb** [comment]
I force pushed the changes. I feel like I messed something up though.

**Comment by yousong** [comment]
Rebase against master branch of this repo and force push again should fix this up.

**Comment by thess** [comment]
Add mdns feature request to MPD in: https://github.com/openwrt/packages/commit/4b5577016cd8f1e4b75a88e5d62e55ec431e5926

@yousong - Thanks for the suggestion to get port number from config file.

@neheb - I suggest you close this and re-submit the **openssh** part separately with a correct description and signed-off-by. Thanks for the suggestion.

**Pull request closed without merging**